src/histolab/tiler.py

```diff
@@ -611,10 +611,16 @@ def _save_report(
             List of the tiles' filename
         """
 
-        report = pd.DataFrame(
-            {"filename": filenames, "score": np.array(highest_score_tiles)[:, 0]}
-        )
-        report.to_csv(report_path, index=None)
+        header = ["filename", "score"]
+        rows = [
+            dict(zip(header, values))
+            for values in zip(filenames, np.array(highest_score_tiles)[:, 0])
+        ]
+
+        with open(report_path, "w+") as f:
+            w = csv.DictWriter(f, fieldnames=header, delimiter=";")
```

Comment:
before the delimiter was `,`, the pandas default. Any particular reason to use `;` ?

Reply:
yes no reason at all ... changed to `,`